recipes/geant4/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "10.5" %}
+
+package:
+    name: geant4
+    version: {{ version }}
+source:
+    url: http://cern.ch/geant4-data/releases/geant4.10.05.tar.gz
+    sha256: 2a86499d8327abc68456e5d7fc0303824e5704322291b331857cf4042286656e
+build:
+    number: '0'
+    skip: True  # [win]
```

Comment:
You have a `bld.bat` but are skipping Windows, is this intentional?

Reply:
I will remove it and leave the Windows build for someone else to add afterwards.

I am currently writing a test and will update the PR later.